autonomy/cli/deploy.py

```diff
@@ -266,13 +266,6 @@ def run_deployment_from_token(  # pylint: disable=too-many-arguments, too-many-l
                 version=DEFAULT_IMAGE_VERSION,
                 push=False,
             )
-            build_image(
```

Comment:
So this was responsible for building deps right?

Reply:
Yes